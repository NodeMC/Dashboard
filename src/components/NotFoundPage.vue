<template>
    <div class="section has-text-centered">
        <p class="is-size-2">There's nothing here.</p>
        <router-link to="/" class="is-size-4">Return to homepage</router-link>
    </div>
</template>

<script>
export default {

};
</script>

<style>

</style>
