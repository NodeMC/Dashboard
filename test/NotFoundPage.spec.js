import { shallow } from "vue-test-utils";
import { createRenderer } from "vue-server-renderer";

import NotFoundPage from "@/components/NotFoundPage";

describe("NotFoundPage.vue", () => {
    it("matches snapshot", () => {
        const renderer = createRenderer();
        const wrapper = shallow(NotFoundPage);
        renderer.renderToString(wrapper.vm, (err, str) => {
            if (err) throw new Error(err);
            expect(str).toMatchSnapshot();
        });
    });
});
