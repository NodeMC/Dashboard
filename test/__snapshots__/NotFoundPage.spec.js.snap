// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage.vue matches snapshot 1`] = `
<div class="section has-text-centered">
    <p class="is-size-2">There's nothing here.</p>
    <router-link to="/" class="is-size-4">Return to homepage</router-link>
</div>
`;
